<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>7</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Manage Personal Profiles</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="profileList">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>441</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="profileGroupBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>187</y>
     <width>291</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Profile Details</string>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labelName">
      <property name="text">
       <string>Profile Name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditName"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelBasal">
      <property name="text">
       <string>Basal Rate (U/hr)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="spinBoxBasal"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelCarb">
      <property name="text">
       <string>Carb Ratio (g/U)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="spinBoxCarb"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="labelCorrection">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Correction Factor
(mmol/L per unit)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDoubleSpinBox" name="spinBoxCorrection"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="labelTarget">
      <property name="text">
       <string>Target Glucose</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QDoubleSpinBox" name="spinBoxTarget"/>
    </item>
   </layout>
   <zorder>labelName</zorder>
   <zorder>lineEditName</zorder>
   <zorder>labelBasal</zorder>
   <zorder>spinBoxBasal</zorder>
   <zorder>labelCarb</zorder>
   <zorder>spinBoxCarb</zorder>
   <zorder>spinBoxCorrection</zorder>
   <zorder>labelTarget</zorder>
   <zorder>spinBoxTarget</zorder>
   <zorder>labelCorrection</zorder>
  </widget>
  <widget class="QPushButton" name="logoButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#logoButton {
    background-color: #005F87;
    color: black;
    font-weight: bold;
    font-size: 16px;
    padding: 6px 12px;
    border-radius: 6px;
    border: 1px solid #007ACC;
}

QPushButton#logoButton:hover {
    background-color: #1E90FF;
}

QPushButton#logoButton:pressed {
    background-color: #007ACC;
    color: white;
}
</string>
   </property>
   <property name="text">
    <string>T</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>207</y>
     <width>111</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="buttonLayout">
    <item>
     <widget class="QPushButton" name="buttonCreate">
      <property name="text">
       <string>Create</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonUpdate">
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonDelete">
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonSelect">
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonSave">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
